script.js

```diff
@@ -1,4 +1,238 @@
+      winCountCom,
+      noOfRounds
+    )}%) <br> You win: ${winCountUser} times (${calWinPercentage(
+      winCountUser,
+      noOfRounds
+    )}%)`;
+  }
+
+  // If program choice is scissors and user inputs paper OR If program choice is stone and user inputs scissors OR If program choice is paper and user inputs stone, output is "The user wins."
+  if (
+    (programChoice == "scissors" && userGuess == "paper") ||
+    (programChoice == "stone" && userGuess == "scissors") ||
+    (programChoice == "paper" && userGuess == "stone")
+  ) {
+    noOfRounds += 1;
+    winCountUser += 1;
+    message = `You choose ${userGuess}. <br> The computer choose ${programChoice}. <br><br> You win! Congratulations! <br><br> Computer wins: ${winCountCom} times (${calWinPercentage(
+      winCountCom,
+      noOfRounds
+    )}%) <br> You win: ${winCountUser} times (${calWinPercentage(
+      winCountUser,
+      noOfRounds
+    )}%)`;
+  }
+
+  // If program choice is scissors and user inputs stone OR If program choice is stone and user inputs paper OR If program choice is paper and user inputs scissors, output is "The program wins."
+  if (
+    (programChoice == "scissors" && userGuess == "stone") ||
+    (programChoice == "stone" && userGuess == "paper") ||
+    (programChoice == "paper" && userGuess == "scissors")
+  ) {
+    noOfRounds += 1;
+    winCountCom += 1;
+    message = `You choose ${userGuess}. <br> The computer choose ${programChoice}. <br><br> You lose! Too Bad! <br><br> Computer wins: ${winCountCom} times (${calWinPercentage(
+      winCountCom,
+      noOfRounds
+    )}%) <br> You win: ${winCountUser} times (${calWinPercentage(
+      winCountUser,
+      noOfRounds
+    )}%)`;
+  }
+
+  // if win percentage of user < win percentage of computer, prompt "you can do better!"
+  if (
+    calWinPercentage(winCountUser, noOfRounds) <
+    calWinPercentage(winCountCom, noOfRounds)
+  ) {
+    return `${message} <br><br> You can do better, ${userName}! <br><br> ${userName}, you can type 'scissors', 'paper' or 'stone' to play another round.`;
+  } else if (
+    // if win percentage of user > win percentage of computer, prompt "you are doing great!"
+    calWinPercentage(winCountUser, noOfRounds) >
+    calWinPercentage(winCountCom, noOfRounds)
+  ) {
+    return `${message} <br><br> You are doing great, ${userName}! <br><br> ${userName}, you can type 'scissors', 'paper' or 'stone' to play another round.`;
+  } else if (
+    // if win percentage of user = win percentage of computer, prompt "keep going!"
+    calWinPercentage(winCountUser, noOfRounds) ==
+    calWinPercentage(winCountCom, noOfRounds)
+  ) {
+    return `${message} <br><br> Keep Going, ${userName}! <br><br> ${userName}, you can type 'scissors', 'paper' or 'stone' to play another round.`;
+  }
+  return message;
+};
+
```

Comment:
Logic is looks correct! maybe you can challenge yourself to refactor this into something shorter since it is very similar to your normal game function? i know  you can do it haha